src/MyMoneroApi.js

```diff
@@ -0,0 +1,300 @@
+    //   scanned_block_height: asNumber,
+    //   scanned_height: asNumber,
+    //   start_height: asNumber,
+    //   total_received: asString,
+    //   transaction_height: asNumber,
+    //   transactions: asArray(asObject({
+    //     coinbase: asBoolean,
+    //     hash: asString,
+    //     height: asNumber,
+    //     id: asNumber,
+    //     mempool: asBoolean,
+    //     mixin: asNumber,
+    //     timestamp: asDate,
+    //     total_received: asString,
+    //     total_sent: asString,
+    //     unlock_time: asNumber
+    //   }))
+    // })
+
+    const parsed = await parserUtils.Parsed_AddressTransactions__async(
+      this.keyImageCache,
+      result,
+      address,
+      privateViewKey,
+      publicSpendKey,
+      privateSpendKey,
+      this.cppBridge
+    )
+    return parsed.serialized_transactions
+  }
+
+  async getAddressInfo(keys: WalletKeys): Promise<BalanceResults> {
+    const { address, privateSpendKey, privateViewKey, publicSpendKey } = keys
+    const result = await this.fetchPostMyMonero('get_address_info', {
+      address,
+      api_key: this.apiKey,
+      view_key: privateViewKey
+    })
+    // const asAddressInfo = asObject({
+    //   blockchain_height: asNumber,
+    //   locked_funds: asString,
+    //   rates: asObject(asNumber),
+    //   scanned_block_height: asNumber,
+    //   scanned_height: asNumber,
+    //   spent_outputs: asArray(
+    //     asObject({
+    //       amount: asString,
+    //       key_image: asString,
+    //       mixin: asNumber,
+    //       out_index: asNumber,
+    //       tx_pub_key: asString
+    //     })
+    //   ),
+    //   start_height: asNumber,
+    //   total_received: asString,
+    //   total_sent: asString,
+    //   transaction_height: asNumber
+    // })
+
+    try {
+      const parsed = await parserUtils.Parsed_AddressInfo__async(
+        this.keyImageCache,
+        result,
+        address,
+        privateViewKey,
+        publicSpendKey,
+        privateSpendKey,
+        this.cppBridge
+      )
+      return {
+        blockHeight: parsed.blockchain_height,
+        totalReceived: parsed.total_received_String,
+        lockedBalance: parsed.locked_balance_String,
+        totalSent: parsed.total_sent_String
+      }
+    } catch (error) {
+      console.log(error)
+    }
+    return {
+      blockHeight: 0,
+      totalReceived: '0',
+      lockedBalance: '0',
+      totalSent: '0'
+    }
+  }
+
+  async createTransaction(
+    keys: WalletKeys,
+    opts: CreateTransactionOptions
+  ): Promise<CreatedTransaction> {
+    const { address, privateSpendKey, privateViewKey, publicSpendKey } = keys
+    const { amount, isSweepTx = false, priority = 1, targetAddress } = opts
+
+    console.log('mymonero', {
+      address,
+      amount: '0',
+      api_key: this.apiKey,
+      app_name: 'test',
+      app_version: 'test',
+      dust_threshold: '2000000000',
+      mixin: 10,
+      use_dust: true,
+      view_key: privateViewKey
```

Comment:
Even if this log doesn't make it to the app log we should probably remove it

Reply:
Note this is only the View key so access to it can't spend funds.

Reply:
The whole comment is misleading now - most of these things have been removed or renamed. I'll just log the `opts`, since that has no keys in it.